### PR TITLE
Accept terms prompt can take a level prop for h tag level

### PR DESCRIPTION
**Pull request by noahgelman**
## Description
Accept terms prompt can take a level prop for h tag level

## Original issue(s)
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/217

## Acceptance criteria
- [ ] Accept terms prompt can take a level prop for h tag level
- [ ] Default to h1 if no level value is passed to keep backward compatibility

### CODE REVIEW

File: src/platform/user/authorization/components/AcceptTermsPrompt.jsx [hunk 1]
```diff
@@ -60,6 +60,8 @@ class AcceptTermsPrompt extends React.Component {
       return <div />;
     }
 
+    const H = `h${this.props.level || 1}`;
```
Comment (ndsprinkle) [inline src/platform/user/authorization/components/AcceptTermsPrompt.jsx:63]: Hey @noahgelman, could you change this variable name to be something a bit more descriptive? Making it something more explanatory would be nice for future development in this section.

Comment (ndsprinkle) [inline src/platform/user/authorization/components/AcceptTermsPrompt.jsx:63]: Hey @noahgelman, could you change this variable name to be something a bit more descriptive? Making it something more explanatory would be nice for future development in this section.

Comment (noahgelman) [reply src/platform/user/authorization/components/AcceptTermsPrompt.jsx:63]: @ndsprinkle Can do!

**Review by ndsprinkle — COMMENTED** [review]
@noahgelman Thanks for updating the variable name :) One more thing to bug you with, could you also add a unit test for this? Probably would just have two scenarios: passing in nothing should render an `h1` and passing in a level `x` should render an `hx`

**Review by ndsprinkle — APPROVED** [review]

**Pull request merged**